stable_baselines3/a2c/a2c.py

```diff
@@ -55,7 +55,7 @@ def __init__(
         self,
         policy: Union[str, Type[ActorCriticPolicy]],
         env: Union[GymEnv, str],
-        learning_rate: Union[float, Callable] = 7e-4,
+        learning_rate: Union[float, Callable[[float], float]] = 7e-4,
```

Comment:
Maybe add that to the type aliases? (as it comes quite often)
for the name, I'm not sure, maybe `LearningRateOrSchedule`